packages/monaco/src/browser/textmate/textmate-snippet-completion-provider.ts

```diff
@@ -0,0 +1,60 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+export class TextmateSnippetCompletionProvider implements monaco.languages.CompletionItemProvider {
```

Comment:
Why the `Textmate` prefix? It is just a template.

Reply:
Not sure, what you are referring to. The snippet structure is textmate specific.

Reply:
When removing it, I changed my mind. I'd prefer keeping the signature.

Reply:
I see, I was confused with this: https://code.visualstudio.com/docs/editor/userdefinedsnippets.